code/src/index.css

```diff
@@ -5,9 +5,125 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  box-sizing: border-box;
+	background-color: black;
+	color: white;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.album-container {
+  display: grid;
+}
+
+.album-image-container {
+  position: relative;
+}
+
+.album-card {
+  width: 300px;
+  padding: 15px;
+}
+
+.header-title {
+  color:white;
+}
+
+.album-title {
+  font-family: Helvetica;
+  font-size: 14px;
+  color: white;
+}
+
+.album-title-link {
+  text-decoration: none;
+}
+
+.album-title-link:hover {
+  text-decoration: underline;
+  text-decoration-color: white;
+}
+
+.icons-container {
+  display: none;
+  position: absolute;
+  justify-content: center;
+  align-items: center;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+}
+
+.album-image-container:hover .icons-container{
+  display: flex;
+  background: rgba(0, 0, 0, 0.7);
+  transition: 0.5s;
+}
+  
+.icon {
+  width: 25px;
+  filter: invert(100%);
+  opacity: 0.7;
+}
+
+.icon:hover {
+  opacity: 1;
+}
+
+.icon-play{
+  width: 60px;
+  filter: invert(100%);
+  padding-left: 20px;
+  padding-right: 20px;
+  opacity: 0.7;
+}
+
+.icon-play:hover {
+  width: 80px;
+  padding-left: 10px;
+  padding-right: 10px;
+  opacity: 1;
+}
+
+
+.artists, .artist-and {
+  font-family: Helvetica;
+  font-size: 14px;
+  color: #a0a0a0;
+  text-decoration: none;
+}
+
+.artists:hover {
+  text-decoration: underline;
+}
+
+/* In case of multiple artists, this adds ', ' before every artist that is not the first nor the last one
+.artist-list > :not(:last-child):not(:first-child):before {
+  content: ", ";
+  white-space: pre;
+  color: #a0a0a0;
+}
+
+/* In case of multiple artists, this adds ' & ' before the last artist*/
+/* .artist-list > :last-child:before {
+  content: " & ";
+  white-space: pre;
+  color: #a0a0a0;
+} */ 
+
+@media (min-width: 640px) {
+  .album-container {
+    grid-template-columns: repeat(2, 1fr);
+    grid-column-gap: 44px;
+  }
+}
+
+@media (min-width: 1444px) {
```

Comment:
Suggestion to add this a bit earlier so that it's the same on any desktop regular/laptop screens to. Maybe have two desktop versions, one for smaller desktop and one for bigger. :)